.buildkite/macos.rayci.yml

```diff
@@ -1,28 +1,6 @@
 group: macos tests
 sort_key: "~macos"
 steps:
-  # block on premerge
```

Comment:
This is temporary change to test out updated test suite in this PR. Will be reverted before merging.